pandas/_version.py

```diff
@@ -48,12 +49,11 @@ class NotThisMethod(Exception):
     pass
 
 
-LONG_VERSION_PY = {}
-HANDLERS = {}
+HANDLERS = {}  # type: Dict[str, Dict[str, Callable]]
 
 
-def register_vcs_handler(vcs, method):  # decorator
-    def decorate(f):
+def register_vcs_handler(vcs: str, method: str) -> Callable:  # decorator
```

Comment:
Glancing at existing code it looks like we could still be more strict about the type here. Any reason to not specify the ReturnType of the callable here? Looks to always be a dict

Reply:
ok, can do that.

Reply:
@WillAyd on a closer look, isn't the method `register_vcs_handler` returning a method(callable) `decorate` which returns another method `f` which is the method which is decorated. And the method `f` is returning `dict` I think.
So I think it will be something like this:

    def register_vcs_handler(vcs: str, method: str) -> Callable[[Callable], Callable[[], Dict[str, str]]]:

And what to do about the args the method `f`(the method which is getting decorated) as it's args might vary. Usually we would do `*args, **kwargs` in these situations but there is no such option in mypy.